templates/deployment-worker.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: worker
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: worker
+        image: eanylin/airflow-helm:1.8.1-1.6.2
```

Comment:
I didn't mean to suggest it wasn't public -- I really meant an att-comdev owned image, that way the whole stack is manageable by the team.